File-Formats/DocIO/word-to-markdown.md

```diff
@@ -0,0 +1,799 @@
+
+You can specify the folder location to export the images using MarkdownExportImagesFolder API.
+
+The following code example illustrates how set images folder to export the images while converting Word document to Markdown file.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+//Open an existing Word document. 
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    //Set images folder to export images. 
+    document.SaveOptions.MarkdownExportImagesFolder = "D:\\WordToMdConversion ";
+    //Save the Word document as a Markdown file.
+    document.Save("WordtoMd.md", FormatType.Markdown);
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Open an existing Word document.
+Using document As WordDocument = New WordDocument("Input.docx", FormatType.Docx)
+'Set images folder to export images. 
+document.SaveOptions.MarkdownExportImagesFolder = "D:\\WordToMdConversion ";
+'Save the document as a Markdown file.
+document.Save("WordtoMd.md", FormatType.Markdown)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//DocIO doesn’t supports MarkdownExportImagesFolder API in UWP and Xamarin platforms
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as a Stream.
+using (FileStream docStream = new FileStream("Input.docx", FileMode.Open, FileAccess.Read))
+{
+    //Load the file stream into a Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Set images folder to export images. 
+        document.SaveOptions.MarkdownExportImagesFolder = "D:\\WordToMdConversion ";
+
+        //Save a Markdown file to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Markdown);
+        outputStream.Position = 0;
+        //Download as a Markdown file in the browser.
+        return File(outputStream, "application/msword", "WordtoMd.md");
+    }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//DocIO doesn’t supports MarkdownExportImagesFolder API in UWP and Xamarin platforms
+{% endhighlight %}
+{% endtabs %}
+
+### Customize the image path
+
+DocIO provides an ImageNodeVisited event, which is used to customize the image path to set in output Markdown file and save images externally while converting Word document to Markdown.
+
+The following code example illustrates how to save Image files during Word to Markdown Conversion.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+//Open an existing Word document. 
+using (WordDocument document = new WordDocument(@"Input.docx"))
+{
+    //Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage;
+    //Save the Word document as a Markdown file.
+    document.Save("WordtoMd.md", FormatType.Markdown);
+}
+The following code examples shows the event handler to customize the image path and save the image in external folder.
```

Comment:
The following code examples show the event handler to customize the image path and save the image in an external folder.